kouta-backend/src/main/scala/fi/oph/kouta/domain/oppilaitos.scala

```diff
@@ -385,15 +403,19 @@ case class OppilaitoksenOsaListItem(oid: OrganisaatioOid,
                                     muokkaaja: UserOid,
                                     modified: Modified)
 
-case class Yhteystieto(osoite: Option[Osoite] = None,
+case class Yhteystieto(nimi: Kielistetty = Map(),
+                       osoite: Option[Osoite] = None,
+                       kayntiOsoite: Option[Osoite] = None,
```

Comment:
Näitä jäin miettimään pitäisikö olla postiOsoite ja kayntiOsoite?

Nyt toki tuohon osoitteeseen menee entuudesta syötetty arvo ilman migraatiota, mutta mitenhän tämä olisi by-the-book oikein? Tallennetussa datassa vilisee niitä postilokeroita joten varmaan tämä olisi nyt kuitenkin oikein.

Konfo-ui:lla näiden priorisoinnin voi tehdä sitten kyllä miten haluaa, eli kartalla näytetään käyntiosoite jos sellainen on annettu, muuten osoite.

Reply:
Kyllähän se tosiaan voisi olla selkeämpi nimetä uusiksi jos tuota käytetään tarkoituksessa postiosoite. Se olisi myös etsittävämpi sana koodista kuin geneerinen osoite. Tässä vaiheessa olisi vielä helppo lisätä migraation alkuun lause joka tekee renamen. Kouta-ui:lle tulisi varmaan tämän myötä muutoksia?

Reply:
Joo, ei ole isoja muutoksia kuitenkaan